tests/test_uwsgi_plugin.py

```diff
@@ -0,0 +1,62 @@
+from testing.sandbox import install_fake_uwsgi
+install_fake_uwsgi()
```

Comment:
Oh yea? Do you think that would be simpler than patching `sys.modules` when all we want to accomplish importing something since we mock out the actual `uwsgi` behavior? I'm open to either approach.